docs/API/fliplet-notifications.md

```diff
@@ -29,9 +29,44 @@ var instance = Fliplet.Notifications.init({});
 
 In-app notifications are displayed in the notifications inbox component of Fliplet Apps. They can optionally include a push notification to be sent to the devices as described further below.
 
+#### Send to all users
+
+Send an in-app notification using the `insert` method:
+
+```js
+// insert a new notification for all users
+instance.insert({
+  status: 'published',
+  data: {
+    title: 'Greetings',
+    message: 'Hi John!'
+  }
+})
+```
+
+#### Add a link to the notification
+
+Add a link to the in-app notification using the `navigate` option:
+
+```js
+// insert a notification linking to a screen with a query
+instance.insert({
+  status: 'published',
+  data: {
+    title: 'Greetings',
+    message: 'Hi John!',
+    navigate: {
+      action: 'screen',
+      page: 123,
+      query: 'weather=sunny'
```

Comment:
doesn't it need the `?` prefix? other parts of the docs seem to have it

Reply:
It doesn't technically need it. Would you prefer it added for consistency?

Reply:
If it doesn't need it then let's leave this without and update the others. Less is better!

Reply:
I think because Studio UI includes the ? too we should include it. At some point they can probably all be removed as it [the question mark is optional](https://github.com/Fliplet/fliplet-api/blob/b5addfb76cb21d8c1e363babb029718a918cda92/public/assets/fliplet-core/1.0/navigate.js#L835).